src/pythermalcomfort/models.py

```diff
@@ -20,6 +21,20 @@
     two_nodes_optimized_return_set,
 )
 
+# import packages for jos-3 model
+import csv
```

Comment:
move this import to the top since are python libraries and then we can import all the files and functions from pythermalcomfort